Container Security/kernel-eks-implementation-guide.md

```diff
@@ -0,0 +1,242 @@
+# Implementation Guide for CrowdStrike Falcon Sensor for Linux on EKS Kubernetes cluster using Helm Chart
+
+This guide works through creation of new Kubernetes cluster, deployment of Falcon Sensor for Linux using Helm Chart, and demonstration of detection capabilities of Falcon Container Workload Protection.
+
+No prior Kubernetes or Falcon knowledge is needed to follow this guide. First sections of this guide focus on creation of EKS cluster, these sections may be skipped if you have access to an existing cluster.
+
+Time needed to follow this guide: 45 minutes.
+
+
+## Pre-requisites
+
+You will need AWS credentials and docker tool installed locally.
+
+## Deployment
+
+### Step 1: Enter the tooling container
+
+ - Install [docker](https://www.docker.com/products/docker-desktop) container runtime
+ - Enter the [tooling container](https://github.com/CrowdStrike/cloud-tools-image)
+   ```
+   docker run --privileged=true \
+       -e FALCON_CLIENT_ID="$FALCON_CLIENT_ID" \
+       -e FALCON_CLIENT_SECRET="$FALCON_CLIENT_SECRET" \
+       -e CID="$CID" \
+       -v /var/run/docker.sock:/var/run/docker.sock \
+       -v ~/.aws:/root/.aws -it --rm \
+       quay.io/crowdstrike/cloud-tools-image
+   ```
```

Comment:
Where do the docs tell the reader where/how to set the $FALCON_CLIENT_* vars? Should these be environmental variables or passed natively into the CLI?